src/NodeBind.js

```diff
@@ -173,7 +173,11 @@
     var eventType = getEventForInputType(input);
 
     function eventHandler() {
-      observable.setValue(input[property]);
+      if (property == 'value' && input.type == 'number' )
+        observable.setValue(input.valueAsNumber);
+      else
```

Comment:
trivial: I don't think the JS style in this file uses dangling else. I'd suggest curlies for the if-else, or maybe restructure:

```js
var isNum = property == 'value' && input.type == 'number';
observable.setValue(isNum ? input.valueAsNumber : input[property]);
```

the JS style guide endorses ternary op for what it's worth: https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Tips_and_Tricks#Tips_and_Tricks